physics/kepler_orbit.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "physics/body.hpp"
+#include "physics/degrees_of_freedom.hpp"
+
+namespace principia {
+namespace physics {
+
+template<typename Frame>
+struct KeplerianElements {
+  double eccentricity;
+  Length semimajor_axis;
+  Angle inclination;
+  Angle longitude_of_ascending_node;
+  Angle argument_of_periapsis;
+  Angle mean_anomaly;
+};
+
+struct TwoBodySystem {
+  not_null<MassiveBody const*> const primary;
+  not_null<Body const*> const secondary;
+};
+
+template<typename Frame>
+class KeplerOrbit {
+ public:
+  KeplerOrbit(TwoBodySystem const& system,
+              Instant const& epoch,
+              KeplerianElements<Frame> const& elements_at_epoch);
+  RelativeDegreesOfFreedom<Frame> StateVectors(Instant const& t) const;
```

Comment:
"state vectors" is a [standard term](https://en.wikipedia.org/wiki/Orbital_state_vectors).
Of one body relative to the other, we don't have much else to relate to.